modin/experimental/core/execution/native/implementations/hdk_on_native/base_worker.py

```diff
@@ -251,4 +251,4 @@ def import_pandas_dataframe(cls, df, name=None):
         str
             Imported table name.
         """
-        return cls.import_arrow_table(pa.Table.from_pandas(df))
+        return cls.import_arrow_table(pa.Table.from_pandas(df), name=name)
```

Comment:
Why was this option not used before?

Reply:
Seems it was missing.

Reply:
Why are you making this change now? Do you have a test that doesn't work without it?

Reply:
In this case, again logically unrelated things :(

Reply:
Do you think a separate PR is required for such a minor and obvious fix?

Reply:
I agree, but this is an internal code and should not affect users.